src/views/profile/followersList.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Moment from "react-moment";
```

Comment:
What does this do?

Reply:
The library helps to Parse, validate, manipulate, and display dates and times in JavaScript.
We use it here to achieve the time ago effect.